### PR TITLE
Check policy tags for ignore/include

### PR DESCRIPTION
**Pull request by andrewjhumphrey**
4439076 (Andrew Humphrey)
   Fetch Policy Tags

   We want to be able to ignore policies as well as roles using our standard
   tagging mechanisms.  Tagging of policies is a relatively new feature, but
   it is in general release.  We need it to support programmatically generated
   things like the roles and policies that Cloudwatch dashboards generates,
   which add a random label at the end of the resource name, but a differently 
   formatted label than cloudwatch uses :-(

6d9d963 (Andrew Humphrey)
   ListPolicyTags needs aws-sdk >= 1.37.10

### CODE REVIEW

File: iamy/aws.go [hunk 1]
```diff
@@ -196,6 +197,16 @@ func (a *AwsFetcher) marshalPolicyDescriptionAsync(policyArn string, target *str
 	}()
 }
 
+func (a *AwsFetcher) fetchPolicyTags(policyArn string, tags *map[string]string) {
+	log.Println("Fetching tags for", policyArn)
```
Comment (simpsora) [inline iamy/aws.go:201]: Do you need a trailing space after the "for" ?

**Review by simpsora — APPROVED** [review]
My untrained Go eye says :+1:

File: iamy/aws.go (hunk 1 above)
Comment (andrewjhumphrey) [reply iamy/aws.go:201]: Nah, go helpfully (sometimes not so much) adds a space between arguments when printing strings. Output looks like:

```
DEBUG 2022/01/25 08:37:05 Fetching tags for arn:aws:iam::909551307430:policy/invoke-initiator-look-up
```

Comment (simpsora) [reply iamy/aws.go:201]: Hooray for Go! (first time that's ever been said, I imagine).

**Pull request merged**